src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -256,6 +259,32 @@ public void testStorageTableMetadataDuringFileIngest() {
 
   }
 
+  @Test
+  public void testEnumerateFileRefEntries() {
+    String targetDirPath = String.format("/%s/", UUID.randomUUID());
+    List<FireStoreDirectoryEntry> fileList =
+        IntStream.range(0, 5)
+            .boxed()
+            .map(
```

Comment:
`mapToObj()` is the same as `.boxed().map()`
```suggestion
            .mapToObj(
```